docs/packages.md

```diff
@@ -24,6 +24,22 @@ The Linux Software Repository can be configured to automatically install the Lin
 > Optionally, if you prefer manual configuration, the Linux Software Repository configuration files are available at [https://packages.microsoft.com/config](https://packages.microsoft.com/config/). The name and location of these files can be located using the following URI naming convention:
 `https://packages.microsoft.com/config/<Distribution>/<Version>/prod.(repo|list)`
 
+### Directory structure
+
+At the root of packages.microsoft.com, you’ll see folders like:
+
+- `/config/<distro>/<version>/prod.list`: Contains repo configuration files for each supported distribution.
+- `/repos/`: Contains product-specific repositories (for example, edge, azure-cli, microsoft-prod).
+- `/ubuntu/`: Contains distribution-specific repos for Microsoft packages that integrate with Ubuntu’s APT system.
```

Comment:
- `/<distro>/<version>/`: Contains distribution-specific repos for Microsoft packages that are built for a particular distribution-version. These are all symlinks / copies of some repositories that can also be found under `/repos/` or `/yumrepos/`, but surfaced at the top level too for convenience.